cmd/down.go

```diff
@@ -24,6 +24,7 @@ import (
 	"github.com/spf13/cobra"
 )
 
+// TODO: comment
```

Comment:
It would be great if someone else do it (ideally authors of those functions).
I just put it there so golint is not complaining and we can merge it and start checking new PRs.